build/templates/rep_caps.rst.mako

```diff
@@ -40,27 +45,43 @@ ${helper.get_rst_header_snippet(name, '-')}
     .. py:attribute:: ${module_name}.Session.${name}[]
 
 % if len(prefix) > 0:
-        If no prefix is added to the items in the parameter, the correct prefix will be added when
-        the driver function call is made.
+        ..
+            If no prefix is added to the items in the parameter, the correct prefix will be added when
+            the driver function call is made.
 
-        .. code:: python
+            .. code:: python
+
+                session.${name}['0-2'].channel_enabled = True
+
+            passes a string of :python:`'${prefix}0, ${prefix}1, ${prefix}2'` to the set attribute function.
 
-            session.${name}['0-2'].channel_enabled = True
+            If an invalid repeated capability is passed to the driver, the driver will return an error.
 
-        passes a string of :python:`'${prefix}0, ${prefix}1, ${prefix}2'` to the set attribute function.
+            You can also explicitly use the prefix as part of the parameter, but it must be the correct prefix
+            for the specific repeated capability.
 
-        If an invalid repeated capability is passed to the driver, the driver will return an error.
+            .. code:: python
 
-        You can also explicitly use the prefix as part of the parameter, but it must be the correct prefix
-        for the specific repeated capability.
+                session.${name}['${prefix}0-${prefix}2'].channel_enabled = True
+
+            passes a string of :python:`'${prefix}0, ${prefix}1, ${prefix}2'` to the set attribute function.
 
 % endif
         .. code:: python
 
-            session.${name}['${prefix}0-${prefix}2'].channel_enabled = True
+            session.${name}[0].channel_enabled = True
+
+        sets :py:attr:`channel_enabled` to :python:`True` for ${name} 0.
+
+        .. code:: python
 
-        passes a string of :python:`'${prefix}0, ${prefix}1, ${prefix}2'` to the set attribute function.
+            session.${name}[0:2].channel_enabled = True
+        
+        sets :py:attr:`channel_enabled` to :python:`True` for ${name} 0, 1, 2.
```

Comment:
If I add anything to the recommend types above, I should add an example usage, here.